journalism_workflow/static/journalism_workflow/copy_editor/js/controllers.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('journalism_workflow.copy_editor.controllers')
+    .controller('CopyEditorController', CopyEditorController)
+
+  CopyEditorController.$inject = ['$scope', 'orchestraService'];
+
+  function CopyEditorController($scope, orchestraService) {
+    var vm = $scope;
+
+    // Store the article text document URL for easier summary later
+    var documentCreationStep = orchestraService.taskUtils.findPrerequisite(
```

Comment:
based on the .get.get.get.get.get above, we should probably have a `find_prerequisite` helper in python as well---worth an issue?